### DISCUSSION TITLE
How do I connect to nextcloud once installed on unraid? Cannot get past "The login is blocked since Nextcloud is running"

### DISCUSSION THREAD

**Discussion by mbc0** [nextcloud/all-in-one#4983] [Questions] [answered]
Hi,

I have managed to install on my unraid server

I have nextcloud_aio_mastercontainer set to port 444 as I cannot use 443 as unraid uses it
I have a cloudflare tunnel set to go to 444 
I have spent approx 6 hours installing/uninstalling due to permission issues, install failures but finally everything seems to be ok but I do not know how to actually connect to the nextcould instance? 444 takes me to the AIO login page and I have tried my lan address with apache port 11000 and it directs me to the same AIO login page but via my cloudflare tunnel.

Any help would be really appreciated!  Sorry, I am not very experienced so apologies if I have missed or misunderstood something!

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Reply by mbc0** [reply]
Hi, Thank you for your response, 
I am wondering if point 10 will be the issue? "If your reverse proxy is configured to use the host network (as recommended in the above docs) or running on the host, make sure that you've configured your firewall to open port 443 and 80"

I cannot use port 443 or 80 as unraid uses these ports, this is why when I had everything working using the standard Nextcloud container I edited the container template so my host used 444 but the container used 443.

Has Nextcloud AIO been successfully tested on unraid?

Also, I tried connecting to the apache port 11000 but there is no response, I can confirm all the dockers are running in unraid

![image](https://github.com/user-attachments/assets/a25fe9ee-8e63-4422-836c-db9e3a3c1bc0)
